docs/roman/data_products/science_products.rst

```diff
@@ -0,0 +1,141 @@
+Science Products
+----------------
+The following sections describe the format and contents of each of the Roman ASDF science
+products.
+
+.. _uncal:
+
+Uncalibrated raw data: ``uncal``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Exposure raw data products are designated by a file name
+suffix of "uncal." These files usually contain only the raw detector pixel values
+from an exposure, with the addition of meta data associated with the exposure. The
+resultantdq array is an optional array used to flag missing data in the data Formatting
+process.
+
++--------------+----------+------------+-------+-------------------------------+
+| data array   |          | Data Type  | Units | Dimensions                    |
++==============+==========+============+=======+===============================+
+|  data        | Required | uint16     | DN    |  ncols x nrows x nresultants  |
```

Comment:
Since this is using numpy arrays directly in Python, as opposed through FITS, should the dimensions be listed as 
`nresultants x nrows x ncols`?

Reply:
If this is the right way to present the dimensions then they need to change in every row in this table.

Reply:
Updated